site/en/articles/new-headless/index.md

```diff
@@ -0,0 +1,204 @@
+
+// …
+
+await driver.quit();
+```
+
+
+See [the Selenium team’s blog post](https://www.selenium.dev/blog/2023/headless-is-going-away/#what-are-the-two-headless-modes) for more information, including examples using other language bindings.
+
+### Headless-specific command-line flags
+
+The following command-line flags are available for the new Headless mode.
+
+#### `--dump-dom`
+
+The `--dump-dom` flag prints the serialized DOM of the target page to stdout. Here’s an example:
+
+```sh
+chrome --headless=new --dump-dom https://developer.chrome.com/
+```
+
+Note that this is different from simply printing the HTML source code (which you might do with `curl`). `--dump-dom` gives you the result of Chrome parsing the HTML code into a DOM, executing any `<script>`s that might alter the DOM — and then turning that DOM back into a serialized string of HTML.
+
+#### `--screenshot`
+
+The `--screenshot` flag takes a screenshot of the target page and saves it as `screenshot.png` in the current working directory. It’s especially useful in combination with the `--window-size` flag. Here’s an example:
+
+```sh
+chrome --headless=new --screenshot --window-size=412,892 https://developer.chrome.com/
+```
+
+#### `--print-to-pdf`
+
+The `--print-to-pdf` flag saves the target page as a PDF named `output.pdf` in the current working directory. Here’s an example:
+
+```sh
+chrome --headless=new --print-to-pdf https://developer.chrome.com/
+```
+
+#### `--timeout`
+
+The `--timeout` flag specifies the delay in milliseconds after which the page’s content is captured by `--dump-dom`, `--screenshot`, and `--print-to-pdf`. When neither `--timeout` nor `--virtual-time-budget` (see below) are specified, the page content is captured as soon as the page is loaded.
+
+To illustrate its use, consider [this demo page which increments, logs, and displays a counter every second](https://mathiasbynens.be/demo/time) using `setTimeout(fn, 1000)`. Here’s the relevant code:
+
+```html
+<output>0</output>
+<script>
+  const element = document.querySelector('output');
+  let counter = 0;
+  setInterval(() => {
+    counter++;
+    console.log(counter);
+    element.textContent = counter;
+  }, 1_000);
+</script>
+```
+
+After one second, the page contains “1”; after two seconds, “2”, and so on. Here’s how you’d capture the page’s state after 5 seconds and save it as a PDF:
+
+```sh
+chrome --headless=new --print-to-pdf --timeout=5000 https://mathiasbynens.be/demo/time
+```
+
+The `--timeout=5000` flag tells Chrome to wait for 5 seconds before printing the PDF. Thus, this process takes at least 5 seconds to run.
+
+#### `--virtual-time-budget`
+
+The `--virtual-time-budget` enables time travel! Well, kinda. Virtual Time acts as a “fast-forward” for any time-dependent code (e.g. `setTimeout`/`setInterval`). It forces the browser to execute any of the page’s code as fast as possible while making the page believe that the time actually goes by.
```

Comment:
```suggestion
The `--virtual-time-budget` enables time travel! Well, to some extent. Virtual Time acts as a “fast-forward” for any time-dependent code (for example, `setTimeout`/`setInterval`). It forces the browser to execute any of the page’s code as fast as possible while making the page believe that the time actually goes by.
```
I like the phrase but that's too informal: https://developers.google.com/style/tone#examples

Reply:
This kinda ruins the joke. See what I did there? Oh well.